src/test/java/uk/ac/cam/cl/dtg/isaac/api/AuthenticationFacadeIT.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.cam.cl.dtg.isaac.api;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+import org.eclipse.jetty.http.CookieCompliance;
+import org.eclipse.jetty.server.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import uk.ac.cam.cl.dtg.segue.api.AuthenticationFacade;
+
+import static org.easymock.EasyMock.*;
+import static org.eclipse.jetty.http.HttpCookie.SAME_SITE_LAX_COMMENT;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({AuthenticationFacade.class})
+@PowerMockIgnore({"javax.xml.datatype.*", "javax.management.*", "javax.crypto.*", "javax.net.ssl.*", "javax.net.*"})
+public class AuthenticationFacadeIT extends IsaacIntegrationTest {
+
+    private AuthenticationFacade authenticationFacade;
```

Comment:
This is being assigned in the `setUp` method, but it does not seem to be used anywhere.